statistics-production/ees.qmd

```diff
@@ -912,15 +918,15 @@ The above code should produce something along the lines of the following plots:
 
 ---
 
-#### Embedding R-Shiny blocks
+#### Embedding R Shiny blocks
 
 ---
 
-If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R-Shiny produced chart.
+If you need to include a type of chart that isn't possible using the standard EES chart options, for example an interactive chart with filters, you can embed a block to display a custom R Shiny produced chart.
 
-To embed a shiny chart, you'll first need to create an R-Shiny application containing the chart using the DfE tiny-shiny template and get it hosted on our DfE GitHub and ShinyApps accounts. More details on both of these are available in the tiny-shiny section of this guidance.
+To embed a shiny chart, you'll first need to create an R Shiny application containing the chart using the DfE tiny-shiny template and get it hosted on our DfE GitHub and ShinyApps accounts. More details on both of these are available in the tiny-shiny section of this guidance.
```

Comment:
Think the first Shiny in this sentence should have big S? Sorry, missed it on the first go

Reply:
Sorted